K2Bridge/Visitors/Aggregations/RangeAggregationVisitor.cs

```diff
@@ -4,8 +4,9 @@
 
```

Comment:
This file was completely changed... i guess since the union part didn't work well with the new approach..

Reply:
Yes. @tomconte updated DateRange and Filters, and I updated Range to use the same pattern.